dell/xps/15-9560/nvidia/default.nix

```diff
@@ -14,9 +14,9 @@
   ##### disable intel, run nvidia only and as default
   hardware.nvidia.prime = {
     # Bus ID of the Intel GPU.
-    intelBusId = lib.mkDefault "PCI:0:2:0";
+    intelBusId = "PCI:0:2:0";
```

Comment:
What's the reasoning behind removing mkDefault?

Reply:
I do not believe that ID changes unless one is using a completely different laptop model. In that case, the user should be warned if they or another module tries to change the value without explicitly forcing it.